README.md

```diff
@@ -4,7 +4,7 @@
 
 ![screenshot](./leaderBoard.png)
 
-> The leaderboard website displays scores submitted by different players. it also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service. Checkout the live site [here](https://daudi13.github.io/LeaderBoard/dist/). The project is to be executed in three phases namely
+> The leaderboard website displays scores submitted by different players. it also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service. Checkout the live site [here](https://daudi13.github.io/LeaderBoard/dist). The project is to be executed in three phases namely
 
 ### stages
 
```

Comment:
- [ ] Kindly update the screenshot in your README with the recent styling version of your Leaderboard.
- [ ] The MIT link is broken. Please fix it so your README is documented professionally.

Reply:
I've also  updated the screenshot and fixed the broken link to my license. Thank you for pointing that out